deis/manifests/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: deis-logger-fluentd
+  labels:
+    heritage: deis
+spec:
+  template:
+    metadata:
+      name: deis-logger-fluentd
+      labels:
+        heritage: deis
+    spec:
+      containers:
+      - name: deis-logger-fluentd
+        image: quay.io/jchauncey/fluentd-kubernetes:syslog
```

Comment:
Yes I want to move these over. Just getting this started.

In deis/manifests/deis-logger-fluentd-daemon.yaml
<https://github.com/deis/charts/pull/27#discussion_r47167767>:

> +apiVersion: extensions/v1beta1
> +kind: DaemonSet
> +metadata:
> +  name: deis-logger-fluentd
> +  labels:
> +    heritage: deis
> +spec:
> +  template:
> +    metadata:
> +      name: deis-logger-fluentd
> +      labels:
> +        heritage: deis
> +    spec:
> +      containers:
> +      - name: deis-logger-fluentd
> +        image: quay.io/jchauncey/fluentd-kubernetes:syslog

Before merge, we should put these in the deisci repo. Same with other
images.

—
Reply to this email directly or view it on GitHub
<https://github.com/deis/charts/pull/27/files#r47167767>.